src/Pages/TwoFactor.php

```diff
@@ -193,11 +193,21 @@ public function enableAction(): Action
                 $formData['email'] = $data['email'];
             }
 
-            if ($this->twoFactorData['option']) {
-                $formData['two_factor_type'] = TwoFactorType::tryFrom($this->twoFactorData['option']);
+            try {
+                if ($this->twoFactorData['option']) {
+                    $formData['two_factor_type'] = TwoFactorType::tryFrom($this->twoFactorData['option']);
+                }
+            } catch (\Exception $e) {
+                Notification::make()
```

Comment:
I think it would be better to show the message in the form itself like a validation error. Can you do that maybe?

Reply:
I've changed the code, check it out.

Reply:
I'm totally new with filamentphp, haven't found a way of returning the error to the form yet, if this new approach isn't the best solution, I will continue to search for.